distributions/raspijamulus.sh

```diff
@@ -22,10 +22,10 @@ fi
 if [ -d "${OPUS}" ]; then
   echo "The Opus directory is present, we assume it is compiled and ready to use. If not, delete the opus directory and call this script again."
 else
-  wget https://archive.mozilla.org/pub/opus/${OPUS}.tar.gz
-  tar -xzf ${OPUS}.tar.gz
-  rm ${OPUS}.tar.gz
-  cd ${OPUS}
+  wget https://archive.mozilla.org/pub/opus/"${OPUS}.tar.gz"
+  tar -xzf "${OPUS}.tar.gz"
+  rm "${OPUS}.tar.gz"
+  cd "${OPUS}"
```

Comment:
Again, no need to quote `${OPUS}`, because we know what is in it, from line 4

Reply:
Isn't the point of populating a variable that you _don't_ have to know its value to use it?  In which case you should make sure you're using it safely.  (And again, if this is a constant, use `readonly` to declare it.)